Week 1/Weekly Practice Problems (Week 1)/0412: Fizz Buzz/Mohammad.cpp

```diff
@@ -0,0 +1,27 @@
+class Solution {
+public:
+    vector<string> fizzBuzz(int n) {
+        vector<string> v;
+        for(int i = 1 ; i <= n ; i++){
+        if((i % 3 == 0) && (i % 5 == 0) ){
+            
+            v.push_back("FizzBuzz");
+        }
+        else if(i % 3 == 0 ){
+            v.push_back("Fizz");
+        }
+        else if(i  % 5 == 0){
+            v.push_back("Buzz");
+
+        }
+        else{
+            stringstream s;
+            s << i;
+            string str = s.str();
+            v.push_back(str);
```

Comment:
We can shorten this a bit by using the to_string() function on i
```suggestion
            v.push_back(to_string(i));
```